optax/transforms/__init__.py

```diff
@@ -80,9 +83,11 @@
     "MaskedState",
     "MultiSteps",
     "MultiStepsState",
+    "multi_transform",
+    "MultiTransformState",
     "named_chain",
     "NonNegativeParamsState",
-    "partition",
+    "partition",    
```

Comment:
Could you delete the empty trailing whitespace here?

Reply:
done :)